fmriprep/config.py

```diff
@@ -510,18 +511,19 @@ def init(cls):
 class seeds(_Config):
     """Initialize the PRNG and track random seed assignments"""
 
+    _random_seed = None
     master = None
-    """Master seed used to generate all other tracked seeds"""
+    """Master random seed to initialize the Pseudorandom Number Generator (PRNG)"""
     ants = None
     """Seed used for antsRegistration, antsAI, antsMotionCorr"""
 
     @classmethod
     def init(cls):
-        cls.master = workflow.random_seed
+        if cls._random_seed is not None:
+            cls.master = cls._random_seed
```

Comment:
Can you elaborate a bit more about this change? It looks more compact than before, but I'm not sure I fully understand what are you trying to improve here.

Reply:
sure, i originally put `random_seed` within the `execution` config class, but this PR shifts it to a private variable within the `seeds` class. Additionally, this check needs to be added when reusing configs to ensure seeds are being reused, while also allowing command line options to override.